src/pages/roadmap.md

```diff
@@ -6,42 +6,43 @@ hide_table_of_contents: true
 
 Timelines are estimates and subject to change. For information about delivered features and fixes, see the [Changelog](/docs/changelog).
 
-## Release cycle
+## 2023 milestones
 
-We typically follow this release cycle:
+### Q4
 
-- Alpha: Super early peek into the feature being built. Not ready for use in production apps.
-- Beta: Opportunity to test, validate, and provide feedback. Not ready for use in production apps.
-- Production: Ready for use in production apps.
+- Allow/denylist user preferences available in all SDKs
 
-## Features roadmap
+## 2024 milestones
 
-### 2023 Q3
+### Q1
 
-- Onchain transaction reference content type
-- React SDK in Production
-- React Native SDK in Production (currently in Beta)
-- Android SDK in Production (currently in Beta)
-- XMTP v3 in Alpha
+- [Group chat via Messaging Layer Security (MLS)](https://github.com/xmtp/roadmap/milestone/1) in alpha
+- [Contact directory via did:ethr](https://github.com/xmtp/roadmap/milestone/5) in alpha
+- [XMTP testnet](https://github.com/xmtp/roadmap/milestone/6) in alpha
 
-### 2023 Q4
+### Q2
 
-- XMTP v3 in Beta
+- [Group chat via MLS](https://github.com/xmtp/roadmap/milestone/2) in production
+- [XMTP testnet](https://github.com/xmtp/roadmap/milestone/7) in production
```

Comment:
What will be the difference between the alpha testnet and production testnet? Who do we intend to have run the production nodes? What is the production use case for devs?